### ISSUE TITLE
App crashes when I select "Project Path" on Mac M1

### ISSUE THREAD

**Issue by kennycason** [libgdx/gdx-liftoff#195]
After having issues with the M1 launcher (addressed in previous issue), I download the latest Jar (`GDX Liftoff 1.12.1.13.jar`).

The app launched as expected, and I was able to get through most of the setup options, however every time I click on "Project Path", the app just crashes with no error or anything. Launching the Jar file from my terminal directly and clicking into "Project Path" resulted in the below error:

```
java -jar GDX\ Liftoff\ 1.12.1.13.jar
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSWindow should only be instantiated on the main thread!'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000182c58540 __exceptionPreprocess + 176
	1   libobjc.A.dylib                     0x0000000182749eb4 objc_exception_throw + 60
	2   CoreFoundation                      0x0000000182c7d88c _CFBundleGetValueForInfoKey + 0
	3   AppKit                              0x00000001863d17c0 -[NSWindow _initContent:styleMask:backing:defer:contentView:] + 188
	4   AppKit                              0x000000018652a6b0 -[NSPanel _initContent:styleMask:backing:defer:contentView:] + 48
	5   AppKit                              0x00000001863d16f8 -[NSWindow initWithContentRect:styleMask:backing:defer:] + 48
	6   AppKit                              0x000000018652a664 -[NSPanel initWithContentRect:styleMask:backing:defer:] + 48
	7   AppKit                              0x00000001870072e4 -[NSSavePanel initWithContentRect:styleMask:backing:defer:] + 124
	8   AppKit                              0x0000000186fe0368 -[NSOpenPanel initWithContentRect:styleMask:backing:defer:] + 156
	9   AppKit                              0x000000018700723c -[NSSavePanel init] + 240
	10  AppKit                              0x0000000187005480 +[NSSavePanel _createPanel] + 64
	11  liblwjgl_nfd.dylib                  0x00000001502be4bc liblwjgl_nfd.dylib + 9404
	12  ???                                 0x0000000113ff8a88 0x0 + 4630481544
	13  ???                                 0x0000000113ff504c 0x0 + 4630466636
	14  ???                                 0x0000000113ff504c 0x0 + 4630466636
	15  ???                                 0x0000000113ff517c 0x0 + 4630466940
	16  ???                                 0x0000000113ff517c 0x0 + 4630466940
	17  ???                                 0x0000000113ff562c 0x0 + 4630468140
	18  ???                                 0x0000000113ff517c 0x0 + 4630466940
	19  ???                                 0x0000000113ff0140 0x0 + 4630446400
	20  libjvm.dylib                        0x0000000103b4b5b0 _ZN9JavaCalls11call_helperEP9JavaValueRK12methodHandleP17JavaCallArgumentsP10JavaThread + 924
	21  libjvm.dylib                        0x0000000103b4a574 _ZN9JavaCalls12call_virtualEP9JavaValueP5KlassP6SymbolS5_P17JavaCallArgumentsP10JavaThread + 320
	22  libjvm.dylib                        0x0000000103b4a640 _ZN9JavaCalls12call_virtualEP9JavaValue6HandleP5KlassP6SymbolS6_P10JavaThread + 100
	23  libjvm.dylib                        0x0000000103c1177c _ZL12thread_entryP10JavaThreadS0_ + 156
	24  libjvm.dylib                        0x0000000103b5e6cc _ZN10JavaThread17thread_main_innerEv + 140
	25  libjvm.dylib                        0x000000010401c020 _ZN6Thread8call_runEv + 200
	26  libjvm.dylib                        0x0000000103ec7610 _ZL19thread_native_entryP6Thread + 344
	27  libsystem_pthread.dylib             0x0000000182b06034 _pthread_start + 136
	28  libsystem_pthread.dylib             0x0000000182b00e3c thread_start + 8
)
libc++abi: terminating due to uncaught exception of type NSException
```

```
➜  java --version
java 21.0.2 2024-01-16 LTS
Java(TM) SE Runtime Environment (build 21.0.2+13-LTS-58)
Java HotSpot(TM) 64-Bit Server VM (build 21.0.2+13-LTS-58, mixed mode, sharing)
```

Let me know if you need any more info, or if I'm just missing something obvious. Thanks!

**Comment by kennycason** [comment]
<img width="1777" alt="image" src="https://github.com/user-attachments/assets/d5a8ad73-2181-4fe7-bff9-723909091d10">

**Comment by crykn** [comment]
This should be fixed by 849f78a. You can find a Jar file for the current snapshot [on the libGDX Discord](https://discord.com/channels/348229412858101762/1273840116322009180/1273864973210353756).

**Comment by kennycason** [comment]
Thank you! I'll test out, and appreciate y'alls work on this!

**Comment by kennycason** [comment]
@crykn The jar you linked worked for me! 🙏

**Comment by tommyettinger** [comment]
This should also be fixed in 1.12.1.14, which I'm releasing now! Unfortunately I have no idea what happens with the .app on M1, so that may still be broken in 1.12.1.14 ... It may take a new Construo release, or maybe there was one I didn't update to. Hopefully the JAR is enough.
